README.md

```diff
@@ -75,6 +77,35 @@ generic with minimal need for project-specific behavior.
 [Docker]: https://docs.docker.com/install/overview/
 
 
+### 2.1.1 git-secrets
+
+[git-secrets] helps to prevent secrets (passwords, credentials, etc.) from being
+submitted into a git repository. See the "installing git-secrets" section of the
+project's README.rst for instructions how to install git-secrets on your OS.
+
+Once installed, git-secrets will need to be configured for your local Azul
+repositories. To configure git-secrets in a single local repository, run:
+
+    cd /path/to/my/repo
+    git secrets --install
+    git secrets --register-aws
+    
+Alternatively, you can configure git-secrets globally for all existing
+repositories and those you will initialize or clone in the future:
+
+    git secrets --install ~/.git-templates/git-secrets
+    git config --global init.templateDir ~/.git-templates/git-secrets
+    git secrets --register-aws --global
+
+Once finished, run this check in all your local Azul repositories to confirm
+that git-secrets is installed and configured correctly.
+
+    cd /path/to/my/repo
+    make check_git_secrets
```

Comment:
Since we need to ensure this for every clone of each of our repositories, I don't think we should use a make target here.

Reply:
>  I don't think we should use a make target here.

A script instead?